tests/test_spark.py

```diff
@@ -35,19 +36,23 @@ def _init_logging():
 HDFS_PACKAGE_NAME='beta-hdfs'
 HDFS_SERVICE_NAME='hdfs'
 SPARK_PACKAGE_NAME='spark'
+TERASORT_JAR='https://downloads.mesosphere.io/spark/examples/spark-terasort-1.0-jar-with-dependencies_2.11.jar'
+TERASORT_MAX_EXECUTOR_CORES=6
```

Comment:
Can we call this `TERASORT_MAX_CORES` instead?  It reads as if it's the max number of cores a single executor will have, but that's not the effect.